Applications/Spire/Source/KeyBindings/KeyBindingsWindow.cpp

```diff
@@ -0,0 +1,18 @@
+#include "Spire/KeyBindings/KeyBindingsWindow.hpp"
+
+using namespace boost::signals2;
+using namespace Spire;
+
+KeyBindingsWindow::KeyBindingsWindow(const KeyBindings& key_bindings,
+  QWidget* parent)
```

Comment:
I'm not sure which indentation is correct, but most often in Spire I see 4 whitespaces on both the continuation line and before the colon.

Reply:
Use two levels of indentation only when there is a need to disambiguate the body of an expression with its "introduction".

In this case the introduction is the initializer, but the body is empty, so there is no need to disambiguate anything. However if the body was not empty then the initializer would need to be double indented to make clear the distinction between the initializer and the body.

Similar thing goes for other introducers such as ```if```, ```for```, etc...